### PR TITLE
OCM-3711 | feat: validate disk size during machinepool creation

### PR DESCRIPTION
**Pull request by marcolan018**
https://issues.redhat.com/browse/OCM-3711
https://issues.redhat.com/browse/OCM-3712
Signed-off-by: marcolan018 <llan@redhat.com>

### CODE REVIEW

File: pkg/interactive/validation.go [hunk 1]
```diff
@@ -187,20 +188,25 @@ func AvailabilityZonesCountValidator(multiAZ bool) Validator {
...
-	// parse it to validate it is a valid unit
-	_, err := ocm.ParseDiskSizeToGigibyte(machinePoolRootDiskSize)
-	if err != nil {
-		return fmt.Errorf("failed to parse machine pool root disk size: %v", err)
+		// parse it to validate it is a valid unit
+		size, err := ocm.ParseDiskSizeToGigibyte(machinePoolRootDiskSize)
+		if err != nil {
+			return fmt.Errorf("failed to parse machine pool root disk size: %v", err)
+		}
+		err = diskValidator.ValidateMachinePoolRootDiskSize(version, size)
```
Comment (oriAdler) [inline pkg/interactive/validation.go:206]: NP
```suggestion
		return diskValidator.ValidateMachinePoolRootDiskSize(version, size)
```

**Review by oriAdler — COMMENTED** [review]
Left a small comment, other than that, LGTM.

File: pkg/interactive/validation.go (hunk 1 above)
Comment (marcolan018) [reply pkg/interactive/validation.go]: @oriAdler 
thanks for the review. Made the refactoring, can you help to review it again?

**Pull request merged**